sql/core/src/test/scala/org/apache/spark/sql/CollationSQLExpressionsSuite.scala

```diff
@@ -996,6 +996,30 @@ class CollationSQLExpressionsSuite
         assert(sql(query).schema.fields.head.dataType.sameType(dataType))
       }
     })
+
+    val tableName = "t_diff_collation"
+    withTable(tableName) {
+      sql(s"CREATE TABLE $tableName (" +
+        s"text STRING COLLATE UTF8_BINARY, " +
+        s"pairDelim STRING COLLATE UTF8_LCASE, " +
+        s"keyValueDelim STRING COLLATE UTF8_BINARY) " +
+        s"USING parquet")
```

Comment:
also, there is an ongoing effort to move such tests to `collations.sql` golden file

please see: https://github.com/apache/spark/pull/47828 and https://issues.apache.org/jira/browse/SPARK-48779

Reply:
Updated, I have moved the related tests to file `collations.sql`.